feature/gnmi/ate_tests/telemetry_port_speed_test/telemetry_port_speed_test.go

```diff
@@ -143,6 +143,7 @@ func (tc *testCase) setupAggregateAtomically(t *testing.T) {
 	}
 
 	agg := d.GetOrCreateInterface(tc.aggID)
+	agg.Type = ieee8023adLag
 	agg.GetOrCreateAggregation().LagType = tc.lagType
```

Comment:
I was going to suggest fixing it elsewhere like here:

https://github.com/openconfig/featureprofiles/blob/d3d4ff7252906141715820ad0c48492f828c9a87/feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go#L161-L163

But looks like this is the only place we missed it. For the sake of consistency, let's reorder the lines similar to other places.

```suggestion
	agg := d.GetOrCreateInterface(tc.aggID)
	agg.GetOrCreateAggregation().LagType = tc.lagType
	agg.Type = ieee8023adLag
```